donkeycar/templates/cfg_complete.py

```diff
@@ -137,7 +137,7 @@
 NUM_LAST_LAYERS_TO_TRAIN = 7        #when freezing layers, how many layers from the last should be allowed to train?
 
 #WEB CONTROL
-WEB_CONTROL_PORT = 8887             # which port to listen on when making a web controller
+WEB_CONTROL_PORT = int(os.getenv("WEB_CONTROL_PORT", 8887))  # which port to listen on when making a web controller
```

Comment:
Can you add that into the `cfg_basic.py` too?

Reply:
Hey this is master and there is no `cfg_basic.py`. I will port this to `cfg_basic.py` in dev in another PR.

Reply:
You're right. Looks good.